relayertest/test.go

```diff
@@ -292,9 +295,18 @@ func testPacketRelaySuccess(
 
 	require.Equal(t, srcInitialBalance-expectedDifference, srcFinalBalance)
 	require.Equal(t, dstInitialBalance+testCoinAmount, dstFinalBalance)
+
+	seq, err := srcChain.GetPacketSequence(ctx, srcTxHash)
```

Comment:
maybe `GetPacketSequence` can be changed to take a *types.TxResponse instead of a hash? For all the uses of it, it looks like the transaction has already been fetched with `Chain.GetTransaction`, so we can avoid the double query.

Reply:
The double query is a good point. I was attempting to keep the interface methods as generic as possible without leaking *types.TxResponse because it is tendermint/cosmos specific. It's perhaps pre-optimization on my part thinking about non-cosmos chains. 

But in a future PR, I'll explore using a type we define in our `ibc` package to limit the queries.

Reply:
Yes, the `*types.TxResponse` return type for the `Chain.GetTransaction` should eventually be replaced with our own generic type, or possibly removed and replaced with more specific queries if there isn't a good one-size-fits-all transaction schema.